### PR TITLE
feat: Homepage added

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added Homepage to the project.

## Files to be checked
- src/components/Banner/Banner.js
- src/pages/home-page/Home.js

## Live: https://deploy-preview-2--electron-ecommerce.netlify.app/

### CODE REVIEW

File: src/pages/home-page/Home.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from "react";
+import { Banner } from "../../components/components";
```
Comment (anik31) [inline src/pages/home-page/Home.js]: suggestion - you can name the components.js file as index.js and then while import you only have to use "../../components"

**Review by anik31 — COMMENTED** [review]
Nice work 👍 Everything looks good, just one suggestion.

File: src/pages/home-page/Home.js (hunk 1 above)
Comment (ParthParolekar) [reply src/pages/home-page/Home.js]: Done

File: src/components/Banner/Banner.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <header className="banner">
+      <img
+        src="https://burgeebyte.in/wp-content/uploads/2021/08/websites-to-download-games.jpg"
+        alt="Banner-image"
+      />
+      <div className="banner-cover"></div>
+      <h1 className="banner-heading">
+        One stop for all of your gaming needs
+        <Link to="/Browse">
```
Comment (foolhardy21) [inline src/components/Banner/Banner.js]: lowercase for `to` value in `<Link>`. that's all.

**Review by foolhardy21 — APPROVED** [review]
added a suggestion. rest is good.

**Comment by ParthParolekar** [comment]
Thank you @anik31 and @foolhardy21 for the reviews

**Pull request merged**